### PR TITLE
updating writer handler docs to include all supported parameters

### PR DESCRIPTION
**Pull request by dusvyat**
## Description

Adds additional detail to writer_handler readme

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: mindsdb/integrations/handlers/writer_handler/README.md [hunk 1]
```diff
@@ -38,6 +38,45 @@ The handler has a number of default parameters set, the user only needs to pass
...
+Supported query parameters for USING syntax are as follows:
+
+[For the Writer LLM API]
+
+- `prompt_template` - this is the template that is used to generate the prompt
+- `writer_api_key` - this is the API key that is used to authenticate with the Writer LLM API
+- `writer_org_id` - this is the organization ID that is used to authenticate with the Writer LLM API
+- `base_url` - this is the base URL that is used to authenticate with the Writer LLM API, optional, if not provided uses org_id and model_id
+- `model_id` - this is the model ID that is used to authenticate with the Writer LLM API
+- `max_tokens` - this is the maximum number of tokens that are used to generate the output, the default is 1024
```
Comment (paxcema) [inline mindsdb/integrations/handlers/writer_handler/README.md:50]: nit: would be great to clarify if this includes the # of input tokens

File: mindsdb/integrations/handlers/writer_handler/README.md [hunk 2]
```diff
@@ -38,6 +38,45 @@ The handler has a number of default parameters set, the user only needs to pass
...
+- `temperature` - this is the temperature that is used to generate the output, the default is 0.0
+- `top_p` - this is the top p that is used to generate the output, the default is 1
+- `stop` - this is a list of stop words that are used to generate the output, the default is an empty list
```
Comment (paxcema) [inline mindsdb/integrations/handlers/writer_handler/README.md:53]: perhaps "used to stop generating tokens after these are generated"?

File: mindsdb/integrations/handlers/writer_handler/README.md [hunk 3]
```diff
@@ -38,6 +38,45 @@ The handler has a number of default parameters set, the user only needs to pass
 
 The other parameters have default values
 
+Supported query parameters for USING syntax are as follows:
+
+[For the Writer LLM API]
```
Comment (paxcema) [inline mindsdb/integrations/handlers/writer_handler/README.md:43]: nit: change to a header, e.g. `##`

**Review by paxcema — APPROVED** [review]
Minor observations, no blockers.

File: mindsdb/integrations/handlers/writer_handler/README.md [hunk 4]
```diff
@@ -38,6 +38,45 @@ The handler has a number of default parameters set, the user only needs to pass
...
+
+- `chunk_size` - this is the number of rows that are ingested at a time, the default is 500
+- `chunk_overlap` - this is the number of rows that are overlapped between chunks, the default is 50
+- `generation_evaluation_metrics` - this is a list of metrics that are used to evaluate the generation task, the default is all of the available metrics
+- `retrieval_evaluation_metrics` - this is a list of metrics that are used to evaluate the retrieval task, the default is all of the available metrics
+- `evaluation_type` - this is the type of evaluation that is run, the default is e2e, this can be set to either `generation` or `retrieval`
+- `n_rows_evaluation` - this is the number of rows that are used to evaluate the model, the default is None, which means all rows are used
+- `retriever_match_threshold` - this is the threshold that is used to determine if the retriever has found a match, the default is 0.7
+- `generator_match_threshold` - this is the threshold that is used to determine if the generator has found a match, the default is 0.8
+- `evaluate_dataset` - this is the dataset that is used to evaluate the model, the default is None, which means the model is not evaluated
```
[the remark this answers is not in the document]
Comment (dusvyat) [reply mindsdb/integrations/handlers/writer_handler/README.md]: @paxcema - updated logic to be clearer - see [here ](https://github.com/mindsdb/mindsdb/blob/5bd9f3b041b611f999e0ac70107c24b26accb202/mindsdb/integrations/handlers/writer_handler/writer_handler.py#L75)

**Pull request merged**